### ISSUE TITLE
A11y_SQLAzureDataStudio_ImportBacpac_Import Bacpac Settings_ScreenReader:Heading level is announced for the labels of the edit box.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9172]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudioBenchmark;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win32;#MAS1.3.1;#JAWS;#NVDA;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.16.0-insider
OS: Windows_NT x64 10.0.18363

**Additional Details:**
Screen reader: NVDA, JAWS
MAS References:MAS1.3.1

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Dacpac extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Data-tier application wizard from Context menu.
8. Select third radio button.
9. Navigate till next button & hit enter.
10. Launch screen reader as NVDA.
11. Navigate to the labels of each edit box using insert+numpad keys(7,4,1)

**Actual:**
On navigating to the labels "file location, target server, target database" of the respective edit box using insert +numpad keys they are announced with a heading level "2".

**Same issue found with JAWS.**

**Same issue in the following page:**
1. Export Bacpac Settings
2. Add new controller

**Expected:**
Heading levels for the labels should not be announced.The names of the labels itself should be announced. 

**User Impact:**
Screen reader users will receive wrong information about the heading structure.

**Attachment link for Reference :**
https://microsoft-my.sharepoint.com/:p:/g/personal/v-stfe_microsoft_com1/EbXU-GkOH4JAoCOm_BM2lJkBhN1ntyRERc-WQOqzsfBbHw?e=FRskVU

**Comment by kisantia** [comment]
@fsteffi why is it not acceptable to announce heading level 2? In this [previous issue](https://github.com/microsoft/azuredatastudio/issues/6072) announcing heading level 2 was added for labels and the issue was closed as fixed.

**Comment by fsteffi** [comment]
@kisantia the previous bug the issue was logged for the heading of the page and the header of the table .Here the issue is logged for the labels of the edit box , these labels are not headings hence they should not be announced as headings.

**Comment by alanrenmsft** [comment]
should be using the same component as #9230

**Comment by sspatil123** [comment]
GithubTags:#Closed;Version 1.42.0 insider
![image](https://user-images.githubusercontent.com/85863999/217266121-58f641ad-0bfc-4d47-9105-aa1a5c559e98.png)

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
